packages/strapi-design-system/src/Select/Option.tsx

```diff
@@ -113,14 +72,62 @@ Option.defaultProps = {
   indeterminate: false,
 };
 
-Option.propTypes = {
-  children: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
-  indeterminate: PropTypes.bool,
-  isChild: PropTypes.bool,
-  multi: PropTypes.bool,
-  selected: PropTypes.bool,
-  startIcon: PropTypes.node,
-  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
-};
-
 Option.displayName = 'Option';
+
+interface CheckMarkProps {
+  selected?: boolean;
+  indeterminate?: boolean;
+}
+
+const CheckMark = styled(Box)<CheckMarkProps>`
+  border: 1px solid
+    ${({ theme, selected, indeterminate }) =>
+      selected || indeterminate ? theme.colors.primary600 : theme.colors.neutral300};
+  height: 18px;
+  width: 18px;
+  background-color: ${({ theme, selected, indeterminate }) =>
```

Comment:
I think you could make this a prop of the component instead to not customize too much CSS

Reply:
Same as other, this will be redone in a later PR so will remember this going forward.